packages/lesswrong/server/resolvers/tagResolvers.ts

```diff
@@ -189,6 +191,26 @@ augmentFieldsDict(Tags, {
       }
     }
   },
+  descriptionHtmlWithAnchors: {
+    resolveAs: {
+      type: "String",
+      resolver: async (tag: DbTag, args: {}, context: ResolverContext): Promise<string> => {
+        const description = tag?.description?.originalContents?.data;
```

Comment:
Does this work if the tag was last edited in markdown? I think it might fail. This will probably be fine for our portal, but it's not obvious to me that LW's portal isn't edited in markdown.